### PR TITLE
Add `operable:trigger` command

### PR DESCRIPTION
**Pull request by christophermaier**
This brings the embedded bundle to parity with `cogctl` with respect
to trigger manipulation.

Fixes: #771

### CODE REVIEW

File: priv/repo/migrations/20160615225508_default_trigger_timeout.exs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+defmodule Cog.Repo.Migrations.DefaultTriggerTimeout do
+  use Ecto.Migration
+
+  def change do
+    alter table(:triggers) do
+      modify :timeout_sec, :integer, null: false, default: 30
```
Comment (kevsmith) [inline priv/repo/migrations/20160615225508_default_trigger_timeout.exs]: How does this timeout differ from Cog's pipeline timeout?

Comment (christophermaier) [inline priv/repo/migrations/20160615225508_default_trigger_timeout.exs]: Unrelated: this is how long a trigger will hang around waiting for the pipeline to complete before finalizing the HTTP response. This is already the default in the code, but we didn't actually have a non-null constraint here in the database; I added it here while I was in the neighborhood.

Comment (kevsmith) [inline priv/repo/migrations/20160615225508_default_trigger_timeout.exs]: Aaah. Got it. Thanks for the clarification.

**Comment by kevsmith** [comment]
👍

**Pull request merged**